sidekick/lib/src/init/init_command.dart

```diff
@@ -15,14 +16,37 @@ class InitCommand extends Command {
     argParser.addOption(
       'cliName',
       abbr: 'n',
+      help: 'The name of the CLI to be created. \n'
+          'The `_cli` prefix will be defined automatically.',
+    );
+    argParser.addOption(
+      'entrypointDirectory',
+      abbr: 'e',
       help:
-          'The name of the CLI to be created \nThe `_cli` prefix will be defined automatically',
+          'The directory in which the CLI entrypoint script should be created.',
+    );
+    argParser.addOption(
+      'cliPackageDirectory',
+      abbr: 'c',
+      help: 'The directory in which the CLI package should be created. \n'
+          'This directory must be within the entrypointDirectory, or '
+          'if the entrypointDirectory is inside a git repository, it must be '
+          'within the git repository. \n'
+          'If this path is given as a relative path and there is a git repository, '
+          'it is resolved relative to the repository root. Else it is resolved '
+          'relative to the entrypointDirectory.',
```

Comment:
I would expect the directory to be resolved from `cwd`, not the `entrypointDirectory`

Reply:
Which one of these options do you mean?

Resolve relative paths ...
1. only relative to `cwd`
2. relative to `repoRoot` and if that doesn't work relative to `cwd`

Reply:
always relative to cwd